pynblint/notebook.py

```diff
@@ -19,7 +19,9 @@ class Notebook(RichRenderable):
     """
 
     def __init__(self, path: Path):
+        self.imported_package = None
         self.path: Path = path
+        self.missing_requiremet: set
```

Comment:
This information is not supposed to be stored here. It will be computed as the result of a linting function dedicated to checking the presence of import statements that do not map to the declared project requirements.